### PR TITLE
add an outputs directory

### LINKED ISSUE #55: Allow for read-only inputs
Currently, the workflow writes tot he same directory that it reads data from. Since the input data (which is the outputs from `workflow.data.preparation`) should be read only, we need to update to write the output files from this process to a separate directory.

https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/blob/7a7a0bb7b5f1a8162cee24d3a0eeea1d4ddb28f5/prepare_pacta_indices.R#L27-L30

cc @jdhoffa

### PR DESCRIPTION
**Pull request by cjyetman**
closes #55

### CODE REVIEW

File: prepare_pacta_indices.R [hunk 1]
```diff
@@ -26,7 +26,22 @@ setwd(transition_monitor_dir)
...
+
+output_dir <- file.path("/outputs")
+stopifnot(dir.exists(output_dir))
+
+system_timestamp <- format(
+  Sys.time(),
+  format = "%Y%m%dT%H%M%SZ",
+  tz = "UTC"
+)
+output_dir <- file.path(output_dir, paste(Sys.getenv("R_CONFIG_ACTIVE"), system_timestamp, sep = "_"))
```
Comment (AlexAxthelm) [inline prepare_pacta_indices.R:38]:
```suggestion
output_dir <- file.path(output_dir, paste(pacta_financial_timestamp, system_timestamp, sep = "_"))
```

This gives us the ability to add config options like `2023_azure` (matching what we have in `workflow.data.preparation`)

**Review by jdhoffa — APPROVED** [review]

**Review by AlexAxthelm — DISMISSED** [review]

**Comment by cjyetman** [comment]
@AlexAxthelm can you give this another pass now that I merged your suggestion and fix a bunch of merge conflicts

**Review by AlexAxthelm — DISMISSED** [review]

**Review by jdhoffa — APPROVED** [review]
I am trusting @AlexAxthelm's review and rubber stamping.
I did not review this myself.

**Comment by cjyetman** [comment]
Merge conflicts on this are seriously messed up now. Might have to start from scratch

**Review by AlexAxthelm — APPROVED** [review]

**Comment by AlexAxthelm** [comment]
README looks good to me

**Comment by jdhoffa** [comment]
Superseded by #77

**Pull request closed without merging**